android/dependencies.gradle

```diff
@@ -22,6 +22,7 @@ ext {
     playServicesMapsVersion = "16.1.0"
     playServicesMeasurementBaseVersion = "16.0.2"
     playServicesAnalyticsVersion = "16.0.4"
+    facebookVersion = "4.39.0"
```

Comment:
Why do we need this new dependency?

Reply:
Logging module, we are logging stuff to facebook.